Documentation/ceph-recover-pod-on-LostNode.md

```diff
@@ -0,0 +1,138 @@
+---
+title: ceph-recover-pod-on-LostNode
```

Comment:
Instead of creating a new doc, how about if we just add a new section to the CSI troubleshooting guide?